contributing.md

```diff
@@ -119,11 +119,19 @@ make docs
 
 ## Releasing
 
+Our current process for publishing a release of python-pachyderm consists of the following steps:
+
+- [ ] Ensure compatibility with the latest protos:
+  - [ ] Make sure `version.json` references the most recent compatible release of Pachyderm
+  - [ ] Run `make src/python_pachyderm/proto/v2` to pull the latest pachyderm code
+  - [ ] Run `make lint` and our test suite (see `contributing.md`)
+- [ ] Ensure that all examples still pass:
+  - [ ] All examples in the `examples/` dir (they must be run manually—`tox example` only runs `examples/opencv`
```

Comment:
nitpicky: can we add a space after the hyphen?

Reply:
Per slack, leaving this part as-is